rmgpy/molecule/group.py

```diff
@@ -1117,18 +1117,25 @@ def updateFingerprint(self):
         isomorphism checks.
         """
         cython.declare(atom=GroupAtom, atomType=AtomType)
-        cython.declare(carbon=AtomType, nitrogen=AtomType, oxygen=AtomType, sulfur=AtomType)
-        cython.declare(isCarbon=cython.bint, isNitrogen=cython.bint, isOxygen=cython.bint, isSulfur=cython.bint, radical=cython.int)
-        
+        cython.declare(carbon=AtomType, nitrogen=AtomType, oxygen=AtomType, sulfur=AtomType, chlorine=AtomType, silicon=AtomType)
+        cython.declare(isCarbon=cython.bint, isNitrogen=cython.bint, isOxygen=cython.bint, isSulfur=cython.bint,
+                       isChlorine=cython.bint, isSilicon=cython.bint, radical=cython.int)
+
         carbon   = atomTypes['C']
         nitrogen = atomTypes['N']
         oxygen   = atomTypes['O']
+        # flourine = atomTypes['F']
```

Comment:
I think we should either go ahead and add fluorine and iodine atom types in this PR, or remove this commented line until we are ready.